tests/playwright/sanity/modules/nested-accordion/nested-accordion-atomic-repeaters.test.ts

```diff
@@ -4,14 +4,13 @@ import WpAdminPage from '../../../pages/wp-admin-page';
 import { addItemFromRepeater, cloneItemFromRepeater, deleteItemFromRepeater } from './helper';
 import _path from 'path';
 
-test.describe( 'Nested Accordion experiment is active @nested-atomic-repeaters', () => {
```

Comment:
Please don't delete the test suite

Reply:
didn't delete, renamed, since we don't have nested atomic repeaters experiment anymore